core/api/src/domain/quiz/config.ts

```diff
@@ -11,10 +11,10 @@ export const QuizzesValue: Record<QuizQuestionId, Satoshis> = {
   moneyEvolution: 2 as Satoshis,
   whyStonesShellGold: 3 as Satoshis,
   moneyIsImportant: 3 as Satoshis,
-  moneyImportantGovernement: 4 as Satoshis,
+  moneyImportantGovernment: 4 as Satoshis,
```

Comment:
I don't think this should be changed. this string is also recorded in the db. 
I think we should add an exemption for this word instead